### ISSUE TITLE
Whitty Mod?

### ISSUE THREAD

**Issue by NailyFromBFB** [luckydog7/Funkin-android#162]
Can you please make the Whitty mod on Android? :>

**Comment by BoyfriendFNF** [comment]
like an actual mod that dosent replace week 1? (because there is one that replaces week 1)

**Comment by NailyFromBFB** [comment]
Yes, but before you make the one that doesn't replace week 1, can you please put the link to the one that does replace week 1? :>

**Comment by DanisLixo** [comment]
he said no
